manifests/p/PascalABCNET/PascalABCNET-stable/3.8.2.3074/PascalABCNET.PascalABCNET-stable.installer.yaml

```
@@ -0,0 +1,14 @@
+# Created using wingetcreate 0.5.0.1
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.1.0.schema.json
+
+PackageIdentifier: PascalABCNET.PascalABCNET-stable

```

```suggestion
PackageIdentifier: PascalABCNET.PascalABCNET
```
Stable isn't needed here as the primary channel